config/environment.js

```diff
@@ -46,6 +46,17 @@ module.exports = function(environment) {
             },
         ],
         FB_APP_ID: process.env.FB_APP_ID,
+        microfeedback: {
+            url: process.env.MICROFEEDBACK_URL,
+            JIRA: {
+                components: {
+                    QuickFiles: '13836',
```

Comment:
This is not needed, send the desired config over to @binoculars, we can inject it at deployment time.

Reply:
I guess the idea was to reduce the required configuration. Only `config.microfeedback.url` needs to be configured.

If it's not configured, i.e. `config.microfeedback.url` is `undefined`, the JIRA information here won't be used.

But if you think this doesn't belong, I can remove it.

Reply:
What is the `'13836'` from? It looks like it's specific to our JIRA. I'd like to keep the configurations as generic as possible.

Generally, I like to use environment variables for local development, but I can override the ember environment at deploy time. Maybe something like `microfeedback.options`